source/developers/how-tos/use-python-coverage-comment.rst

```diff
@@ -0,0 +1,38 @@
+How to Set Up Github Action to Monitor Python Coverage
+######################################################
+
+Most of our public repos use `CodeCov`_ to track our coverage, which is a separate service
+that can show how a particular commit changes the impact of an individual PR on the
+overall coverage of a repository. But sometimes, we do not want to use a separate service,
+because the particular repository is private or for some other reason.
+
+For Python repositories in these cases, we recommend using a GitHub action called `python-coverage-comment`_.
+
+
+.. _CodeCov: https://codecov.io
+.. _python-coverage-comment: https://github.com/py-cov-action/python-coverage-comment-action
+
+
+Features of the Action
+**********************
+
+This action will add a comment to PRs indicating how the current PR will affect coverage and how much of
+the current PR's lines have been covered.
+
+It will also create a branch to track the coverage of the default branch over time. This branch will
+also contain generated coverage HTML reports of the default branch.
+
+Setting Up a Repository
+***********************
+
+To enable this action you will need to do a few steps:
+
+1. Add the ``coverage`` library as a dependency for the repository. By default, we do not install this via cookiecutters.
```

Comment:
Which requirements layer will this generally be added to? I'm guessing ci.in or something. Probably should give at least some approximate guidance here, even if repos vary in how they arrange things.